var/spack/repos/builtin/packages/py-bluepyopt/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class PyBluepyopt(PythonPackage):
+    """Bluebrain Python Optimisation Library"""
+
+    homepage = "https://github.com/BlueBrain/BluePyOpt"
+    pypi = "bluepyopt/bluepyopt-1.9.27.tar.gz"
+
+    # NOTE : while adding new release check pmi_rank.patch compatibility
+    version("1.14.4", sha256="7567fd736053250ca06030f67ad93c607b100c2b98df8dc588c26b64cb3e171c")
+
+    # patch required to avoid hpe-mpi linked mechanism library
+    patch("pmi_rank.patch")
+
+    variant("neuron", default=True, description="Use BluePyOpt together with NEURON")
+    variant("scoop", default=True, description="Use BluePyOpt together with py-scoop")
```

Comment:
I usually default to False to match `pip install` behavior and minimize build times

Reply:
I wanted to avoid breakage with behaviour that existed internally for a long time, but I've been informed that it shouldn't cause trouble.